nixos/modules/services/misc/sdrplay.nix

```diff
@@ -0,0 +1,15 @@
+{ config, lib, pkgs, sdrplay, ... }:
+with lib;
+{
+  options.services.sdrplay_api = {
+    enable = mkEnableOption "SDRPlay API service";
+  };
+
+  config = mkIf config.services.sdrplay_api.enable {
+    systemd.services.sdrplay_api = {
```

Comment:
Does the package contain any `systemd` units, or explain how the software should be run? As is this still run the software as the `root` user - something we're generally not fond of in NixOS, especially when it is proprietary software.

Reply:
There is indeed some `systemd` unit inside the `.run` package from SDRplay. Interestingly it explicitly says `User=root`, but it seems the service runs just fine without any privileges. So i changed it to `DynamicUser = true`, which is like "dynamic user with no permissions" it I understand that mechanism correctly.

Reply:
Looks awesome! Bonus points awarded for use of `DynamicUser` 🎉